tests/test-extensions/heisenberg-extension/src/main/java/org/mule/test/heisenberg/extension/HeisenbergOperations.java

```diff
@@ -170,6 +170,11 @@ public PersonalInfo getInlinePersonalInfo(@ParameterGroup(name = "Personal Info
     return info;
   }
 
+  @MediaType(TEXT_PLAIN)
+  public String transform(String transformation) {
```

Comment:
I'm sure this broke some of the tests in `JavaDeclarationDelegateTestCase` because of  `assertTestModuleOperations`. 
Always run all the tests both in `extensions-support` and `extensions-spring-support`